getusermedia.html

```diff
@@ -845,15 +845,15 @@ <h4>Life-cycle</h4>
         <em>end</em> when the source of the track is disconnected or
         exhausted.</p>
         <p>If all {{MediaStreamTrack}}s that are using the same
-        source are [= track/ended =], the source will be
+        source are [= MediaStreamTrack/ended =], the source will be
         [= source/stopped =].</p>
         <p>After the application has invoked the {{MediaStreamTrack/stop()}}
         method on a {{MediaStreamTrack}} object, or once the [=source=] of a
         {{MediaStreamTrack}} permanently ends production of live samples to its tracks,
         whichever is sooner, a {{MediaStreamTrack}} is said to be
-        <dfn id="track-ended" data-dfn-for="track" data-export>ended</dfn>.</p>
+        <dfn id="track-ended" data-dfn-for="MediaStreamTrack" data-dfn-type="dfn" data-export>ended</dfn>.</p>
```

Comment:
```suggestion
        <dfn data-dfn-for="MediaStreamTrack" class="export">ended</dfn>.</p>
```

Reply:
the id is used in other specs, I see no reason to break it; the `dfn-type` is needed to distinguish it from the IDL attribute of the name and scope
```suggestion
        <dfn id="track-ended" data-dfn-for="MediaStreamTrack" data-dfn-type="dfn" class=export>ended</dfn>.</p>
```

Reply:
Hmm... is it defined twice for the two kinds?

Reply:
once per kind, yes